src/lib/y2firewall/widgets/pages/startup.rb

```diff
@@ -44,35 +44,17 @@ def label
         # @macro seeCustomWidget
         def contents
           VBox(
-            status_widget.widget,
+            status_widget,
             VStretch()
           )
         end
 
-        # @return [Symbol, nil] returns :swap_mode if the service is started
-        #   or stopped and returns nil othwerwise
-        def handle(input)
-          result = status_widget.handle_input(input["ID"])
-          return :swap_mode if result == :start || result == :stop
-
-          nil
-        end
-
-        def store
-          system_service.start_mode = status_widget.enabled_flag? ? :on_boot : :manual
-          system_service.reload if status_widget.reload_flag?
-        end
-
-        def help
-          status_widget.help
-        end
-
       private
 
         # This is a generic widget in SLE15; may not be appropriate.
         # For SLE15-SP1, use CWM::ServiceWidget
         def status_widget
-          @status_widget ||= ::UI::ServiceStatus.new(system_service.service)
```

Comment:
method comment looks outdated

Reply:
Yep, that is true. @dgdavid if you plan to improve it then take this into account ;)

Reply:
Yes, I noticed it and I take it in mind. Anyway, thank you both, for the comment and the ping. That way, I shoul not forgot it ;)